packages/core/src/components/modal/test/prevent-true/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Modal - prevent=true</title>
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, minimum-scale=1.0"
+    />
+    <link rel="stylesheet" href="../../../../../dist/tegel/tegel.css" />
+    <script type="module">
+      import { defineCustomElements } from '../../../../../loader/index.es2017.js';
+      defineCustomElements();
+    </script>
+  </head>
+
+  <body>
+    <tds-button id="my-modal-button" data-testid="tds-modal-button-testid" text="Open Modal"></tds-button>
+    <tds-modal header="This is a header" data-testid="tds-modal-testid" size="lg" id="my-modal-button" show prevent="true">
+      <span slot="body">
+        'Where you can put anything you want!'
+        <span slot='actions' class='tds-u-flex tds-u-gap2'>
+          <tds-button data-dismiss-modal size="md" text="Delete" variant="danger"></tds-button>
+          <tds-button data-dismiss-modal size="md" text="Cancel" variant="secondary"></tds-button>
+        </span>         
```

Comment:
```suggestion
      <span slot="body">
        'Where you can put anything you want!'
        </span>
        <span slot='actions' class='tds-u-flex tds-u-gap2'>
          <tds-button data-dismiss-modal size="md" text="Delete" variant="danger"></tds-button>
          <tds-button data-dismiss-modal size="md" text="Cancel" variant="secondary"></tds-button>
        </span>         
```

Missing closing span for body slot.